src/wrap-store/wrapStore.js

```diff
@@ -19,8 +19,9 @@ const promiseResponder = (dispatchResult, send) => {
       });
     })
     .catch((err) => {
+      console.error(err);
```

Comment:
good call. Is there any way to show the call stack as well? throwing an error will show the call stacks in the console, but `console.error` takes that away.